demo/src/screens/componentScreens/ChipScreen.tsx

```diff
@@ -1,13 +1,90 @@
+import _ from 'lodash';
 import React, {Component} from 'react';
 import {Alert} from 'react-native';
-import {Chip, Colors, Spacings, Text, Typography, View} from 'react-native-ui-lib';
+import {Chip, Colors, Spacings, Text, Typography, View, Dialog, WheelPickerDialog} from 'react-native-ui-lib';
 
 const avatarImage = {
   uri: 'https://randomuser.me/api/portraits/women/24.jpg'
 };
 const checkmark = require('../../assets/icons/check-small.png');
+const chevron = require('../../assets/icons/chevronDown.png');
+
 
 export default class ChipScreen extends Component {
+
+  colors = [
+    {value: Colors.red10, label: 'Red'},
+    {value: Colors.blue10, label: 'Blue'},
+    {value: Colors.purple10, label: 'Purple'},
+    {value: Colors.green10, label: 'Green'},
+    {value: Colors.yellow10, label: 'Yellow'}
+  ];
+
+  state = {
+    showDialog: false,
+    selectedValue: this.colors[2].label
+  };
+
+  toggleDialog = (showDialog: boolean) => {
+    this.setState({showDialog});
+  };
+
+  openDialog = () => {
+    this.toggleDialog(true);
+  }
+
+  closeDialog = () => {
+    this.toggleDialog(false);
+  };
+
+  onSelect = (itemValue: string) => {
+    const values = _.filter(this.colors, {value: itemValue});
+    if (values.length > 0) {
+      this.setState({selectedValue: values[0].label});
+    }
+    this.closeDialog();
+  };
+
+  renderItem = ({item: color}) => {
+    return (
+      <Text text50 margin-20 color={color.value}>
+        {color.label}
+      </Text>
+    );
+  };
+
+  renderContent = () => {
+    const {selectedValue} = this.state;
+
+    return (
+      <WheelPickerDialog
```

Comment:
Why did you add a picker to the chip example screen? 
Don't you think it's making the example screen a bit more complicated than needed? 

What do you want to demonstrate here?

Reply:
That was a request by ux

Reply:
ohh ok :/